tracing-flame/src/lib.rs

```diff
@@ -0,0 +1,277 @@
+//! A Tracing [Layer][`FlameLayer`] for generating a folded stack trace for generating flamegraphs
+//! and flamecharts with [`inferno`]
```

Comment:
Can the data be fed into `infero` as the application is running, or is this data only written on the application's exit? Can you clarify which it is in the documentation?

Reply:
I think in theory they could use a channel that implements writer on one and and reader on the other end and pass the writer into FlameLayer and the reader into Inferno, so probably.

Reply:
I'm gonna try to add an example using `UnixSocket`s to show how this could be implemented

Reply:
Does inferno work with streaming input?

Reply:
No idea, I'll ask jon